### PR TITLE
Add et.json

### PR DESCRIPTION
**Pull request by MrMightyGamer**
Did most of this, some words couldn't be translated or I just didn't know the translation. If something is wrong, feel free to contact me.

MrMightyGamer#1337

### CODE REVIEW

**Comment by MrMightyGamer** [comment]
I translated twitter-line1, but I honestly have 0 idea how to undo the en.json file removal.

File: et.json [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            "title-all" : "Kõik esemed",
+            "error" : "Ei suutnud esemeid saada, palun proovi uuesti mõne hetke pärast",
+            "error-too-long" : "See võtab kauem aega, kui eeldatud"
+        },
+        "shop" : {
+            "title" : "Praegune vahetus",
+            "title-featured" : "Esile toodud",
+            "title-daily" : "Igapäevane",
+            "error" : "Pood pole saadaval, palun proovi uuesti mõne hetke pärast",
+            "line1" : "Praegune eseme poe vahetus Fortnite Battle Royale's - uueneb iga päev kella [00:00 UTC](https://time.is/UTC|tooltip=Current time in UTC|newtab=true) ajal.",
```
Comment (Krymonota) [inline et.json]: Please translate the tooltip ("Current time in UTC").

**Comment by MrMightyGamer** [comment]
Nevermind, figured it out. I did what you requested, hope there are no more problems :)

File: et.json [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+            "line2" : "Võid näha eilset eseme poodi [siin](/shop/yesterday|tooltip=Yesterday's Item Shop).",
```
Comment (Krymonota) [inline et.json]: Please translate the tooltip ("Yesterday's Item Shop").

**Comment by MrMightyGamer** [comment]
done :)

**Comment by MrMightyGamer** [comment]
Sure thing, I'll do that later when I get home

**Comment by MrMightyGamer** [comment]
Done :) Hope I didn't mess up anything

File: et.json [hunk 3]
```diff
@@ -112,7 +112,28 @@
...
+     "response" : { 
+         "success" : "Hääl salvestatud", 
+         "timeout" : "Taotlus võttis liiga kaua aega ning aegus", 
+         "internal" : "Sisemine Viga", 
+         "failover" : "Taotlus [server]'ile ebaõnnestus, proovib uuesti vaikimisi...", 
+         "ratelimit" : "Liiga palju taotlusi, palun proovi uuesti mõne hetke pärast", 
+         "unavailable" : "Hääletamine pole saadaval kuni eseme pood muutub järgmise minuti jooksul", 
+         "conflict" : "Hääletamine pole võimalik, palun proovi hiljem uuesti.", 
+         "bad-choice" : "Häält ei tunnustatud", 
+         "no-data" : "Andmeid pole saadaval - palun proovi uuesti mõne hetke pärast"
```
Comment (Krymonota) [inline et.json]: ...until here.

File: et.json [hunk 4]
```diff
@@ -112,7 +112,28 @@
...
+         "timeout" : "Taotlus võttis liiga kaua aega ning aegus", 
+         "internal" : "Sisemine Viga", 
+         "failover" : "Taotlus [server]'ile ebaõnnestus, proovib uuesti vaikimisi...", 
+         "ratelimit" : "Liiga palju taotlusi, palun proovi uuesti mõne hetke pärast", 
+         "unavailable" : "Hääletamine pole saadaval kuni eseme pood muutub järgmise minuti jooksul", 
+         "conflict" : "Hääletamine pole võimalik, palun proovi hiljem uuesti.", 
+         "bad-choice" : "Häält ei tunnustatud", 
+         "no-data" : "Andmeid pole saadaval - palun proovi uuesti mõne hetke pärast" 
+       } 
+       }
```
Comment (Krymonota) [inline et.json]: Please add 5 spaces at the beginning of the line.

File: et.json [hunk 5]
```diff
@@ -112,7 +112,28 @@
...
+         "success" : "Hääl salvestatud", 
+         "timeout" : "Taotlus võttis liiga kaua aega ning aegus", 
+         "internal" : "Sisemine Viga", 
+         "failover" : "Taotlus [server]'ile ebaõnnestus, proovib uuesti vaikimisi...", 
+         "ratelimit" : "Liiga palju taotlusi, palun proovi uuesti mõne hetke pärast", 
+         "unavailable" : "Hääletamine pole saadaval kuni eseme pood muutub järgmise minuti jooksul", 
+         "conflict" : "Hääletamine pole võimalik, palun proovi hiljem uuesti.", 
+         "bad-choice" : "Häält ei tunnustatud", 
+         "no-data" : "Andmeid pole saadaval - palun proovi uuesti mõne hetke pärast" 
+       }
```
Comment (Krymonota) [inline et.json]: Please add 9 spaces at the beginning of the line.

**Comment by MrMightyGamer** [comment]
Added the spaces as requested, everything should be fine now, I hope :P

**Comment by Krymonota** [comment]
Thank you 👍🏼 
Looks good now and can be merged. 

PS: You may want to edit your first post and include your Discord discriminator (`Username#ID`) so that you definitely get your translator role on the Discord server and your ad-free account on the website.

File: et.json [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+                "dropdown" : "kasutaja",
+                "account" : "kasutaja",
+                "logout" : "logi välja"
+            },
+            "languagePicker" : {
+                "trigger" : "keel",
+                "title" : "Vali keel",
+                "currentLanguage" : "Praegu kasutate [language]",
+                "error" : "Tekkis viga keele faili tõmbamisel",
+                "authors" : "Tõlkis: MrMightyGamer#1337",
```
Comment (Krymonota) [inline et.json]: Please remove your name (` MrMightyGamer#1337`). I don't think this string is currently in use, but it should automatically append the name of each author, possibly also for other languages.

File: et.json [hunk 7]
```diff
@@ -0,0 +1,296 @@
...
+                "hour-singular" : "tund",
+                "hours-plural" : "tundi"
+            },
+            "yesterday" : {
+                "title" : "Eilne eseme poe vahetus"
+            }, 
+            "vote" : { 
+                "title" : "Mis sa arvad sellest poest?", 
+                "line1" : "See lähtestub iga päev kella 00:00 UTC ajal.", 
+                "line2" : "Vaadake uuesti üle meie [privacy policy](/privacy) enne hääletamist",
```
Comment (Krymonota) [inline et.json]: Please translate `privacy policy`.

File: et.json [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+                    "title" : "Hävita Konto",
+                    "line1" : "Kui sa enam ei soovi enda kontot, võid konto ja selle andmed eemaldada [siin](/account/destroy)"
+                }
+            },
+            "oauth" : {
+                "title" : "Logi sisse fnbr.co'sse",
+                "discord-button" : "Logi sisse Discord'iga",
+                "legacy-text" : "Teisel juhul, kui sul on legaat konto aastast 2018, mis vajab liitumist:",
+                "twitter-button" : "logi sisse Twitter'iga",
+                "privacy-text" : "Logides sisse, lubad meil kasutada teie kasutajaga seotud andmeid, nagu on mainitud meie puutumatusnormide kirjelduses](/privacy)."
```
Comment (Krymonota) [inline et.json]: The left square bracket (`[`) is missing which breaks the link to the privacy policy. You may want to check the English language file for reference.

**Comment by MrMightyGamer** [comment]
There, did everything. Didn't notice the round brackets after privacy policy so I didn't translate that :P

**Comment by Krymonota** [comment]
Thank you! 👍
As far as I can tell, it looks good now and can be merged.

**Comment by MrMightyGamer** [comment]
Alright, it's nice to hear that there are no more problems :P

**Pull request merged**